datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -184,16 +126,19 @@ def test_non_datadog_distribution():
         (
             "datadog-active-directory",
             "1.10.0",
-            "datadog_checks_downloader/datadog_checks/downloader/data/repo/targets/"
             "simple/datadog-active-directory/datadog_active_directory-1.10.0-py2.py3-none-any.whl",
         ),
     ],
 )
 @pytest.mark.skipif(PY2, reason="tuf builds for Python 2 do not provide required information in exception")
 @mock.patch("time.time", mock.MagicMock(return_value=_LOCAL_TESTS_DATA_TIMESTAMP))
-@pytest.mark.skip(reason="currently failing since offline metadata was generated with v9 ceremony but currently on v10")
-def test_local_download(capfd, distribution_name, distribution_version, target):
+def test_local_download(capfd, distribution_name, distribution_version, target, monkeypatch):
     """Test local verification of a wheel file."""
+    monkeypatch.setattr(
+        'datadog_checks.downloader.download.ROOT_LAYOUTS',
+        {'core': '4.core.root.layout', 'extras': '1.extras.root.layout'},
```

Comment:
Why `4.core.root.layout` and not `5.core.root.layout`?

Reply:
`5.core.root.layout` is the original value of that constant already. The test repo used to test locally ("offline") doesn't have a `5.core.root.layout` file, so without patching this constant, this test fails because it can't find that file.

I'm still far from familiar with how the whole TUF/in-toto system we have here works and how it's designed, but I can make a couple of alternative suggestions based on:
https://github.com/DataDog/integrations-core/blob/a8b01ea5ab8d2528c9f9f44d5ac30db734a0eb5f/datadog_checks_downloader/datadog_checks/downloader/download.py#L139-L142

- Update the data used to populate the local repo every time we update `ROOT_LAYOUTS` with a new version. This is consistent with the hardcoding of this constant, and it could be necessary if we make some change in the code that addresses some change in the new root layout version.
- Avoid hardcoding this and make it possible for the downloader to _auto-discover_ what version it needs.

Reply:
Note for future reference: we decided in an out-of-band discussion that we'll update the tarball for the local repo to be consistent with the current online version of the repo, removing the need to patch this in the test (and will probably simplify a couple of other things). I'll update this PR with the necessary changes once I get that done.

Reply:
+1, let's have a sync call once you will be available (I've just created one).

If anyone wants to join, let us know. We could provide a script that would automate the tarball creation.

Reply:
@fridex I would like to join! 🤚

Reply:
Added you to the meeting, Andrew 👍🏻  Looking forward to it! (The invite is editable if someone else should be added.)